kie-api/src/main/java/org/kie/api/task/TaskLifeCycleEventListener.java

```diff
@@ -53,7 +55,7 @@
     public void afterTaskSuspendedEvent(TaskEvent event);
     public void afterTaskForwardedEvent(TaskEvent event);
     public void afterTaskDelegatedEvent(TaskEvent event);
-    public void afterTaskNominatedEvent(TaskEvent event);    
+    public void afterTaskNominatedEvent(TaskEvent event);     
```

Comment:
There are unnecessary empty spaces at the line end. Just a tip - not sure what IDE you use; e.g. IntelliJ Idea can be configured to remove the spaces from line ends automatically before file save.